cypress/integration/jsGrowthTableDOM.spec.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+Cypress.Commands.add('multiClick',
+(selector) => {
+  for (let n = 0; n < 7; n++) {
+    cy.get(selector).click( {force: true} );
+  }
+});
+
+describe('Table', () => {
```

Comment:
have we tested that clicking on add row / add column actually adds a row/column?